vividus-plugin-json/src/main/java/org/vividus/json/steps/JsonSteps.java

```diff
@@ -363,16 +368,24 @@ private void executeStepsForFoundJsonElements(String json, String jsonPath, Comp
     public void assertValueByJsonPath(String json, String jsonPath, String comparisonRule, Object expectedData)
     {
         getDataByJsonPathSafely(json, jsonPath, true).ifPresent(jsonByPath -> {
+            List<String> equalAndNotEqualComparisonRules = Arrays.asList("IS_EQUAL_TO", "IS_NOT_EQUAL_TO");
```

Comment:
😮  why strings? why not `List.of`? why not constant or even simple `if` condition?

Reply:
thank you, changed to constants